tests/trainer/test_optimizers.py

```diff
@@ -311,25 +348,125 @@ def test_configure_optimizers_with_frequency(tmpdir):
     model = EvalModelTemplate()
     model.configure_optimizers = model.configure_optimizers__multiple_optimizers_frequency
 
-    trainer = Trainer(
-        default_root_dir=tmpdir,
-        max_epochs=1
-    )
+    trainer = Trainer(default_root_dir=tmpdir, max_epochs=1)
+    result = trainer.fit(model)
+    assert result
+
+
+def test_configure_optimizers_with_frequency_and_lr_schedulers(tmpdir):
```

Comment:
I'd suggest merging the assertions here with both the tests below since it just checks the `opt_idx`. You can access `lr_schedulers` using `trainer.lr_schedulers.`